pkg/messages/baseMessage.go

```diff
@@ -7,15 +7,12 @@ import "github.com/google/uuid"
 type MessageType int
 
 //go:generate go run golang.org/x/tools/cmd/stringer -type=MessageType
-
 const (
 	AskFoodTaken MessageType = iota + 1
 	AskHP
-	AskFoodOnPlatform
 	AskIntendedFoodIntake
 	StateFoodTaken
 	StateHP
-	StateFoodOnPlatform
 	StateIntendedFoodIntake
 	StateResponse
 	ProposeTreaty
```

Comment:
Hmmm I think this breaks the messages logger which assumes 14 message types (bad code design sorry)

Reply:
If it's too annoying to fix then let's just close this PR

Reply:
just realized we need it. if message graphs are fine with now warnings let's merge it